eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -167,12 +167,13 @@ jobs:
           Directory: sdk/${{ parameters.ServiceDirectory }}
           CheckLinkGuidance: $true
 
-      - ${{ each artifact in parameters.Artifacts }}:
-        - template: /eng/common/pipelines/templates/steps/verify-changelog.yml
-          parameters:
-            PackageName: ${{artifact.name}}
-            ServiceName: ${{parameters.ServiceDirectory}}
-            ForRelease: false
+      - ${{if ne(parameters.SDKType, 'data')}}:
```

Comment:
A lot of track 1 libraries don't have change logs and we already shipped those checks in the release steps so I'm skipping them here as well.